data/remnant/remnant missions.txt

```diff
@@ -3445,3 +3445,280 @@ event "remnant: pantica void sprites"
 event "remnant: cognizance timer"
 	system "Nenia"
 		fleet "Ember Waste Node" 6000
+
+
+
+mission "Remnant: Expanded Horizons Astral 1"
+	name "Prep for an expedition to Sagittarius A*"
+	description "To prepare for an expedition, you need to go to <destination> and have 6 space available for a new outfit."
+	blocked "You need at least 1 free bunk to take this mission."
```

Comment:
Because this is a mission that involves ferrying a person around, there is a possibility that it is blocked because of lack of passenger space. I hope this message isn't invoked very often, but it is possible. 

It is also preventative for the second mission in the set, as it is made slightly more likely to have bunk problems because the outfit that is given in the `on complete` section actually requires 1 crew to operate. If the player is flying a penguin, it will mean that if they have *any* other passenger mission active, they will run into trouble in the next mission. So people that are hauling James or Katya or someone from one of the larger mission sets around for the fun of it are going to have to decide what they are going to do.

Reply:
But for a spaceport offered starting mission? Imo, if the player doesn't have the space, it's fine if they don't get offered it.

Reply:
Sure, that makes sense.